app/services/analytics.rb

```diff
@@ -109,6 +109,8 @@ def browser_attributes
   IDV_CANCELLATION = 'IdV: cancellation visited'.freeze
   IDV_CANCELLATION_CONFIRMED = 'IdV: cancellation confirmed'.freeze
   IDV_COME_BACK_LATER_VISIT = 'IdV: come back later visited'.freeze
+  IDV_DOC_AUTH_SUBMITTED_FORM = 'IdV: doc auth form submitted'.freeze
```

Comment:
should we make it clear this is the images stuff? ex `doc auth images form` ?

Reply:
> should we make it clear this is the images stuff? ex `doc auth images form` ?

👍  Updated in e6a9a52cd2fb0d5ee607abddf85149aefaa7af10 to include "image upload" in form and vendor submission constants.